src/routes/OpenAPI/workload-openapi-spec.yaml

```diff
@@ -0,0 +1,272 @@
+openapi: 3.0.2
```

Comment:
Can you move this to src/api-spec.
Also can you rename this to workload-api-spec.yaml?